src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -386,6 +407,13 @@ export default class ManualBusinessInfo extends Mixins(CountriesProvincesMixin,
     }
   }
 
+  /**
+   * Called when a Director's Affidavit is uploaded.
+   */
+  updateBusiness (updatedBusiness: any): void {
+    this.business = updatedBusiness
```

Comment:
Does the store have to be updated also?

Make some changes on this page and then go to the review page and see if your changes are visible.

Reply:
It's updated. Yes, those changes are visible.
![image](https://github.com/bcgov/business-create-ui/assets/122495122/161a1a82-fbec-4b63-a6c2-fccb38fc455f)

Reply:
This comment is probably obsolete. Please verify and resolve if everything works as expected.